packages/cli/bin/designsystemet.ts

```diff
@@ -38,14 +38,17 @@ function makeTokenCommands() {
     .option('-b1, --brand1 <number>', `Brand1 hex color`)
     .option('-b2, --brand2 <number>', `Brand2 hex color`)
     .option('-b3, --brand3 <number>', `Brand3 hex color`)
-    .option('-f, --font-family <string>', `Font family`)
+    .option('-f, --font-family [string]', `Font family`, 'Inter')
+    .option('--theme <string>', `Theme name`)
     .action(async (opts) => {
       // const out = typeof opts.out === 'string' ? opts.out : './dist/tokens';
-      console.log(`Creating tokens with options ${chalk.green(JSON.stringify(opts))}`);
+      console.log(`Creating tokens with options ${chalk.green(JSON.stringify(opts, null, 2))}`);
       const family = typeof opts.fontFamily === 'string' ? opts.fontFamily : 'Inter';
```

Comment:
I think, if you change line 41 to `.option('-f, --font-family <string>', `Font family`, 'Inter')`, you can remove the conditional, as the Commander option type is then no longer `boolean|string`, and the default value is `'Inter'`.

Reply:
When you have an "optional option-argument" (defined by brackets`[string]`), the type is still `string|true` even if you have a default defined.

But thats maybe the wrong option type to to use so I'll change it to "required option-argument"

Reply:
This is incorrect. According to the docs, `[string]` means `boolean|string` and `<string>` means "just" `string`. It has nothing to do with optional or required, which is defined through `.option` or `requiredOption` respectively

Reply:
It should be correct now :)

I wrote option type in the last sentence, but was supposed to be argument type.

Only `--write` is "optional option-argument" now. ie, you can use it as `--write` or with argument `--write "./my-design-tokens"`

Docs is clear on this once you understand the difference between option and option argument 😅 
<img width="1011" alt="image" src="https://github.com/user-attachments/assets/0f26f3e2-c58e-4c8b-928d-55ed9f0a766a">